frontend/cypress/integration/featureFiles/service_details_multicluster.feature

```diff
@@ -46,6 +45,10 @@ Feature: Kiali Service Details page for remote cluster
     When user selects a trace
     Then user sees span details
 
+  Scenario: Don't see tracing info after selecting a trace
+    And user is at the details page for the "service" "bookinfo/ratings" located in the "east" cluster
+    Then user see no traces
```

Comment:
The user sees no traces because there's no traffic for this service in the `east` cluster?

Reply:
Never mind I read your comment:
> Also, I've modified the tests to validate there are not Traces where the workload/service/app from the specific cluster doesn't have traffic